themes/neopolitan/index.html

```diff
@@ -261,37 +265,40 @@
                             <br>
                             <!-- Action -->
                             <% if (locals.action) { %>
-                            <center>
-                                <table style="margin: 0 auto; padding: 0 15px;" cellpadding="0" cellspacing="0" width="100%">
+                                <% action.forEach(function (actionItem) { -%>
+
+                                    <center>
+                                        <table style="margin: 0 auto; padding: 0 15px;" cellpadding="0" cellspacing="0" width="100%">
+                                        <tr>
+                                            <td style="padding: 0 15px;">
+                                            <br>
+                                            <%- actionItem.instructions %>
+                                            <br>
+                                            <br>
+                                            </td>
+                                        </tr>
+                                        </table>
+                                    </center>
+                                    </td>
+                                </tr>
                                 <tr>
                                     <td style="padding: 0 15px;">
+                                    <div><!--[if mso]>
+                                        <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="http://" style="height:50px;v-text-anchor:middle;width:200px;" arcsize="8%" stroke="f" fillcolor="#178f8f">
+                                        <w:anchorlock/>
+                                        <center>
+                                        <![endif]-->
+                                            <a target="_blank" href="<%- actionItem.button.link %>"
+                                    style="background-color:<%- actionItem.button.color %>;border-radius:4px;color:#ffffff;display:inline-block;font-family:Helvetica, Arial, sans-serif;font-size:16px;font-weight:bold;line-height:50px;text-align:center;text-decoration:none;width:200px;-webkit-text-size-adjust:none;">
+                                    <%- actionItem.button.text %>
+                                    </a>
+                                        <!--[if mso]>
+                                        </center>
+                                        </v:roundrect>
+                                    <![endif]--></div>
                                     <br>
-                                    <%- action.instructions %>
                                     <br>
```

Comment:
Another visual bug, too much padding in neopolitan:

![screen shot 2016-10-01 at 11 38 14 pm](https://cloud.githubusercontent.com/assets/4394189/19016898/368fd83e-8830-11e6-8130-f8a1c82ee757.png)

The two `<br>` statements should be moved to line 303 -- outside the `<% if (locals.action) { %>` closure.